content/docs/1.5.0/deploy/upgrade/_index.md

```diff
@@ -11,11 +11,11 @@ There are no deprecated or incompatible changes introduced in v{{< current-versi
 
 # Upgrade Path Enforcement
```

Comment:
Please also update the title to `Upgrade Path Enforcement and Downgrad Prevention`